server/dataset/termdb.test.ts

```diff
@@ -28,86 +28,218 @@ reason:
 
 copyDataFilesFromRepo2Tp()
 
-export default <Mds3>{
-	isMds3: true,
-	cohort: {
-		db: {
-			file: 'files/hg38/TermdbTest/db'
-		},
-		allowedChartTypes: ['summary', 'survival', 'matrix', 'sampleScatter', 'cuminc', 'dataDownload'],
-		termdb: {
-			displaySampleIds: true, // allow to display sample-level data
+export default function () {
```

Comment:
why does it need to export a function?

Reply:
It's used for both genomes (hg38 and hg38-test) in serverconfig.json